profile/config.go

```diff
@@ -278,7 +279,7 @@ Examples:
 		Order:           cfgOptionBlockScopeLANOrder,
 		OptType:         config.OptTypeInt,
 		ExternalOptType: "security level",
-		DefaultValue:    status.SecurityLevelOff,
+		DefaultValue:    status.SecurityLevelsNormalAndExtreme,
```

Comment:
shouldn't that be `SecurityLevelsHighAndExtreme`? Does `SecurityLevelsNormalAndExtreme` even make sense to exist?

Reply:
Yes, this should in fact be `SecurityLevelsHighAndExtreme`.
The Portmaster actually refused to start, because the default did not match the regex! Just did not build and run it ;)
Currently, `SecurityLevelsNormalAndExtreme` does not make sense, but exists historically.